docs/kqueen.rst

```diff
@@ -422,12 +425,27 @@ For more information, see `Issues <https://github.com/Azure/AKS/issues/3>`_, and
 To manually add an existing Kubernetes cluster:
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-1. Log in to the KQueen web UI.
-2. Create ``Manual Provisioner``.
-3. In the ``Create Cluster`` tab choose predefined manual provisioner and attach a valid Kubernetes configuration file.
+#. Log in to the KQueen web UI.
+#. Create ``Manual Provisioner``.
+#. In the ``Create Cluster`` tab choose predefined manual provisioner and attach a valid Kubernetes configuration file.
+
 As a result, the Kubernetes cluster will be attached in a read-only mode.
 
 
+Kubernetes cluster operations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once you successfully provision a Kubernetes cluster, you can manage it through the KQueen web UI.
```

Comment:
successfully provisioned?

Reply:
Once you successfully provision - that's correct